.github/workflows/ci.yml

```
@@ -35,248 +25,186 @@ jobs:
       - uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)